### PR TITLE
fix: fast-pr sandbox fetch 

### PR DESCRIPTION
**Pull request by wangyantong2000**
## Brief Information

This pull request is in the type of ([more info](https://github.com/angular/angular/blob/main/CONTRIBUTING.md#type) about types):

- [ ] build
- [ ] ci
- [ ] docs
- [ ] feat
- [x] fix
- [ ] perf
- [ ] refactor
- [ ] test

Related issues ([all available keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)):

## Details

Previously, every time an iframe was created, the remote file would be pulled again. Now caching has been added, reducing the cost of requesting resources.
And modified the plugin introduction.https://github.com/hypertrons/hypertrons-crx/issues/930
## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/hypertrons/hypertrons-crx/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/hypertrons/hypertrons-crx)

### CODE REVIEW

File: src/pages/ContentScripts/features/fast-pr/index.tsx [hunk 1]
```diff
@@ -55,22 +60,48 @@ const init = async (matchedUrl: MatchedUrl | null) => {
...
-    iframe.contentWindow.postMessage({ command: 'matchUrl', url: url }, '*');
-  }
+  checkCacheAndInit(url);
 }
+
 window.addEventListener('message', (event: MessageEvent) => {
+  if (event.data && event.data.matchedFun) {
+    const matchedFun = event.data.matchedFun;
+    const currentTime = Date.now();
+    localStorage.setItem(CACHE_KEY, JSON.stringify({ matchedFun, timestamp: currentTime }));
```
Comment (frank-zsy) [inline src/pages/ContentScripts/features/fast-pr/index.tsx:103]: Why do you set the `timestamp` to current time for both scenarios? I think we only need to update the cache if the `matchedFunc` is returned with newly fetch.

If so, the config will not be updated if a user keep using the cache without a gap more than 1 hour.

Comment (wangyantong2000) [reply src/pages/ContentScripts/features/fast-pr/index.tsx:102]: Modified and improved.

**Review by Xsy41 — APPROVED** [review]
Good work!

**Review by frank-zsy — APPROVED** [review]
LGTM

**Comment by wangyantong2000** [comment]
Please @Xsy41  release a new version after this PR is merged.

**Pull request merged**